pkg/kubelet/kubelet.go

```diff
@@ -110,6 +111,18 @@ const (
 	housekeepingPeriod = time.Second * 2
 
 	etcHostsPath = "/etc/hosts"
+
+	// Capacity of the channel for recieving pod lifecycle events. This number
+	// is a bit arbitrary and may be adjusted in the future.
+	plegChannelCapacity = 1000
+
+	// TODO(yujuhong): migrate some of the parameters to kubelet flags.
+	// Relisting is used to discover missing container events.
+	// Use a shorter period because generic PLEG relies on relisting for
+	// discovering container events.
+	plegRelistPeriod = time.Second * 3
```

Comment:
I am okay with not exposing this as a flag. It is a bit similar to the resyncInterval, which we expose now though. 

Relisting period will always be a concern unless we adopt docker/rkt event stream completely. But even then, the relisting period would affect container runtimes that don't have an event stream.